property_tests/arbitraries/candid/constructed/variant_arb/definition_arbs.ts

```diff
@@ -120,3 +124,14 @@ function generateCandidTypeObject(
         )
         .join(',')}})`;
 }
+
+function generateAzleCandidTypeObject(fields: Field[]): CandidType {
+    const azleVariantConstructorObj = fields.reduce<{
+        [key: string]: CandidType;
+    }>((acc, [fieldName, fieldDefinition]) => {
```

Comment:
Static type the reduce method itself, get rid of the type argument, no mutations